utils/discord-actions.js

```diff
@@ -1,4 +1,5 @@
 const jwt = require("jsonwebtoken");
+const config = require("config");
```

Comment:
same?

Reply:
Same error for `discord-actions.js`
https://github.com/Real-Dev-Squad/website-backend/blob/develop/utils/discord-actions.js#L3